svc-composition-service/app/proto/auth/odim_auth_pb2.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do not add compiled proto files to github, it should be handled during deployment

Reply:
removed python proto buf files from github. They are generated during build process.